test/stop/criterion_kernels.cpp

```diff
@@ -30,49 +30,48 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ******************************<GINKGO LICENSE>*******************************/
 
-#include <ginkgo/core/stop/criterion.hpp>
-
-
 #include <gtest/gtest.h>
 
 
+#include <ginkgo/core/stop/criterion.hpp>
 #include <ginkgo/core/stop/iteration.hpp>
 
 
-namespace {
+#include "core/test/utils.hpp"
```

Comment:
unused
```suggestion
```

Reply:
curious how you caught that one. Did you use a tool? It's definitely not obvious!

Reply:
I just trust clion on these kinds of things